napari/_qt/_qapp_model/qactions/_window.py

```diff
@@ -0,0 +1,70 @@
+"""Defines window menu actions."""
+
+from typing import List
+
+from app_model.types import Action
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....utils.translations import trans
+from ...qt_main_window import Window
+
+
+def _show_console(window: Window):
+    window._qt_viewer.dockConsole.setVisible(
+        not window._qt_viewer.dockConsole.isVisible()
+    )
+
+
+def _show_layer_controls(window: Window):
+    window._qt_viewer.dockLayerControls.setVisible(
+        not window._qt_viewer.dockLayerControls.isVisible()
+    )
+
+
+def _show_layer_list(window: Window):
```

Comment:
```suggestion
def _toggle_layer_list(window: Window):
```
Based on function code, I suggest updating the name